kasa/smart/modules/homekit.py

```diff
@@ -0,0 +1,40 @@
+"""Implementation of homekit module.
+
+Only function is to indicate homekit Support
+"""
+
+from __future__ import annotations
+
+from ...feature import Feature
+from ..smartmodule import SmartModule
+
+
+class HomeKit(SmartModule):
+    """Implementation of homekit module.
+
+    Currently only adds a feature to indicate the device supports homekit.
```

Comment:
```suggestion
```
I'd also remove this, as the code is straight-forward and this will avoid stale docstrings (this already being stale :-D).

Reply:
Done